app/views/users/_notifications.html.erb

```diff
@@ -55,6 +55,16 @@
     </div>
   </div>
 
+  <div class="crayons-card mb-6 grid gap-6 p-6">
+    <header>
+      <h3 class="mb-2">General Notification Settings</h3>
+    </header>
+    <div class="crayons-field crayons-field--checkbox">
+      <%= f.check_box :article_reaction_notifications, class: "crayons-checkbox" %>
+      <%= f.label :article_reaction_notifications, "Send notifications when someone reacts to my comments or posts", class: "crayons-field__label" %>
```

Comment:
```suggestion
      <%= f.check_box :reaction_notifications, class: "crayons-checkbox" %>
      <%= f.label :article_reaction_notifications, "Send notifications when someone reacts to my comments or posts", class: "crayons-field__label" %>
```

Since the description for this attribute includes both comments _and_ posts, I think that `reaction_notifications` might be a better name for this column, rather than the more specific `article_reaction_notifications`. That name makes it a bit more clear that the attribute applies to _all kinds_ of "reaction" notifications, IMO. Let me know what you think about the idea to rename this column :)

Reply:
Yep, makes sense to me.